plugins/out_http/http.c

```diff
@@ -99,6 +99,33 @@ static char *msgpack_to_json(char *data, uint64_t bytes, uint64_t *out_size)
     /* Format to JSON */
     ret = flb_msgpack_raw_to_json_str(tmp_sbuf.data, tmp_sbuf.size,
                                       &json_buf, &json_size);
+
+    /* Optionally convert to JSON stream from JSON array */
+    if (ctx->out_format == FLB_HTTP_OUT_JSON_STREAM) {
+        char *p;
+        char *end = json_buf + json_size;
+        int level = 0;
+        int in_string = FLB_FALSE;
+        int in_escape = FLB_FALSE;
+
+        for (p = json_buf; p!=end; p++) {
+            if (in_escape)
```

Comment:
what's the function/role of in_escape ?

Reply:
This skips checking *p if the previous character was an escape character.  This addresses the case of:
`[{"foo":"bar\""},{"fizz":"buzz\""}]`

Reply:
thanks